src/Database/PostgreSQL/Transact/Cluster.hs

```diff
@@ -116,16 +122,7 @@ runReadOnly run (CDBT task) conn =
     q = run $ runReaderT (unDBT task) conn
 
 
-instance ExecutionMode 'ReadOnly 'ReadOnly where
-    runSerializable ClusterConnPool{readReplicaConns} task =
-        control $ \run ->
-            withResource readReplicaConns (runReadOnly run task)
-    runNoTransaction ClusterConnPool{readReplicaConns} (CDBT task) =
-        control $ \run ->
-            withResource readReplicaConns (run . runDBTNoTransaction task)
-
-
-instance ExecutionMode 'ReadWrite 'ReadOnly where
+instance ExecutionMode modeConn 'ReadOnly where
     runSerializable ClusterConnPool{readReplicaConns} task =
         control $ \run ->
             withResource readReplicaConns (runReadOnly run task)
```

Comment:
In `master`, we have both `instance ExecutionMode 'ReadOnly 'ReadOnly` and `instance ExecutionMode 'ReadWrite 'ReadOnly`, but these seem to have the same body, so we can remove one of the instances.

In our code-base using `pg-transact-cluster`, we have a bunch of places where we explicitly need to add a constraint like:

```haskell
func :: ExecutionMode mode 'ReadOnly :: CDBT mode m a
```

but this change allows us to remove this constraint, since it can now be solved immediately.